http-client/src/main/java/io/airlift/http/client/jetty/DefaulHttpClientLogger.java

```diff
@@ -88,6 +89,12 @@ public void close()
         asyncAppender.stop();
     }
 
+    @Managed
```

Comment:
This class doesn't seem to be exported. Probably add a `@Managed @Nested getLogger()` to `JettyHttpClient`.